### PR TITLE
add accessibiliy question about color in graph

### PR DESCRIPTION
**Pull request by YuriDevAT**
## Summary of changes

I added a question for the accessibility quiz regarding color in graph to convey meaning.

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/accessibility-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,19 @@
 const accessibilityQuiz = [
+  {
+    Question:
+      "When creating charts, what should be added so that color is not the only identifier to convey the meaning of the information?",
+    Answer:
+      "Patterns.",
+    Distractor1:
+      "Hint Buttons.",
+    Distractor2:
+      "Gradients.",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:10]: periods are not necessary here

```suggestion
    Answer:
      "Patterns",
    Distractor1:
      "Hint Buttons",
    Distractor2:
      "Gradients",
```

File: src/data/accessibility-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+    Distractor3:
+      "There is no need to add an additional identifier, color is enough.",
+    Explanation:
+      "Colorblind and low vision users have difficulties to perceive the color differences. In addition, when the graph appears in greyscale (e.g. user settings, print), it is impossible to every user to see differences.",
+    Link: "https://dequeuniversity.com/tips/color-to-convey-meaning"
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:15]: we only accept links from freeCodeCamp resources, w3.org or wikipedia.
please change the link to one of those resources

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/accessibility-quiz.ts (hunk 1 above)
Comment (YuriDevAT) [reply src/data/accessibility-quiz.ts:10]: Will change that, thanks!

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**